canonical-web-and-design/cn.ubuntu.com
# Make primary_cta optional, remove old takeovers

## Done

- Translations
- Upgrade to discourse v3
- Remove old takeovers

## QA

- Go to https://cn-ubuntu-com-498.demos.haus and fill in a bunch of forms
- Check that the validation is now translated into Chinese
- Check that everything works as intended (you can use ubuntu.com as comparison)
- `primary_cta` is optional now, so if you don't need cta buttons you can leave it blank.
- See if there are any missing engage pages or takeovers [here](https://discourse.ubuntu.com/t/engage-pages-and-takeovers-for-cn-ubuntu-com/19117)

## Issue / Card

#494
